pkg/datainjectionframework/data/dif_metric.go

```diff
@@ -0,0 +1,45 @@
+package data
+
+type CDPMetric struct {
```

Comment:
Would it be more aligned to rename "CDP" to "DIF" for all these defs?
Also, would it be more concise to use the acronym "dif" as the package name instead of fully spelling out the long "datainjectionframework"?